signer/core/cliui.go

```diff
@@ -241,9 +243,28 @@ func (ui *CommandlineUI) OnApprovedTx(tx ethapi.SignTransactionResult) {
 	}
 }
 
+func (ui *CommandlineUI) showAccounts() {
+	accounts, err := ui.api.ListAccounts(context.Background())
+	if err != nil {
+		log.Error("Error listing accounts", "err", err)
+		return
+	}
+	if len(accounts) == 0 {
+		fmt.Printf("No accounts found\n")
+		return
+	}
+	var out = new(strings.Builder)
+	fmt.Fprint(out, "\n------- Available accounts -------\n")
+	for i, account := range accounts {
+		fmt.Fprintf(out, "%d. %s at %s\n", i, account.Address, account.URL)
+	}
+	fmt.Printf(out.String())
```

Comment:
```suggestion
	fmt.Print(out.String())
```
satisfies linter

Reply:
Argh, I think I fixed it now though.. :)